src/chainstate/stacks/db/blocks.rs

```diff
@@ -2321,10 +2337,14 @@ impl StacksChainState {
         )
         .map_err(Error::DBError)?;
 
+        /*
         // garbage-collect
+        // NOTE: temporarily disabled so we can keep invalid microblocks around for further
+        // analysis
```

Comment:
If `delete_microblock_data` is updated to move the microblock data to another table, this should also be uncommented.